src/js/app.js

```diff
@@ -688,7 +688,9 @@ function notify(msg, info, pub) {
   function shouldNotify() {
     if (msg.time < loginTime) { return false; }
     if (info.selfAuthored) { return false; }
-    if (document.visibilityState === 'visible') { return false; }
+    //if (document.visibilityState === 'visible') { return false; }
```

Comment:
Whatsapp and discord don't show notifications when the application is visible, so maybe we shouldn't either?